packages/earthdata-login-client/EarthdataLoginClient.js

```diff
@@ -1,41 +1,22 @@
 'use strict';
 
 const got = require('got');
-const moment = require('moment');
 const { URL } = require('url');
 
-const OAuth2 = require('./OAuth2');
```

Comment:
Tech Debt nit:  Should we consider removing the OAuth2 superclass from google auth as part of this?

Reply:
Done.